package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "book-store",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.0.1"
+  },
```

Comment:
- [ ] It would be better to install `react-redux` to follow the requirements. 😉 

![requirements-react-redux](https://user-images.githubusercontent.com/79876959/144924669-b767cf0d-8036-481b-8d4b-21eb52424846.PNG)